bradford-gill/frillsformer
# Implement Scaled Dot Product

## Description
add multihead attention (MHA) & attention layers layers

## Changes Made

- [ ] Added `Attention` layer for Transformer architecture.
- [ ] Implemented multihead attention 
- [ ] Test new implementations 

## Checklist

Please ensure the following before submitting the PR:

- [ ] PR is Squashed
- [ ] I have added/updated relevant documentation (e.g., README.md, inline docstrings).
- [ ] I have written tests for my changes.

## How to Test


1. Example: Run `pytest tests/` to ensure all tests pass.
2. Example: Execute the sample script in `examples/` directory.

## Notes for Reviewers
bruh attention is fuccced
